sdk/python/feast/feature_store.py

```diff
@@ -1939,6 +1941,9 @@ def serve_transformations(self, port: int) -> None:
 
         transformation_server.start_server(self, port)
 
+    def _refresh_go_server(self):
```

Comment:
this name is pretty misleading - I think your idea is that setting `self._go_server = None` will force the FS to reinitialize the go server on a new `get_online_features`, if so maybe add a comment to describe exactly what you're doing

Reply:
Renamed into `_teardown..`. Yeah, the idea is to kill current instance (if it exists) and it will be recreated on the next requests. Added comment about that